scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -296,4 +296,5 @@ export const generateRelatedAssetIndex = async () => {
   )
 
   console.info('generateRelatedAssetIndex() done')
+  return
```

Comment:
Not returning from this means problems re: exiting process

Reply:
i don't understand why this would be so, but guess it's just one extra return

Reply:
Same thing here, 99% sure we don't need this but just turbo paranoia haha